host-bmc/host_pdr_handler.cpp

```diff
@@ -975,6 +977,7 @@ void HostPDRHandler::setHostSensorState()
 {
     sensorIndex = stateSensorPDRs.begin();
     _setHostSensorState();
+    std::cout << "All the state sensor reading post Reset/Reload is complete\n";
```

Comment:
"Completed get_state_sensor_reading commands on all the host sensors"

Reply:
this does not seem to be the correct - place

Reply:
done